fastlane/Fastfile

```diff
@@ -32,6 +32,24 @@ platform :ios do
     end
   end
 
+  lane :build_simulator do |options|
+    xcversion(version: "~> 11.2")
+
+    cocoapods(repo_update: ENV['MA_FL_CP_REPO_UPDATE'] || false)
```

Comment:
Mini app flag cp(copy) repo update?

Reply:
Cocoapods repo update. Actually I'm not sure why that flag is there. I jus copied it from the cocoapods command in the `ci` script above. The TravisCI script already does a repo update for cocoapods.